iam/sc-enduser-iam.yaml

```diff
@@ -92,23 +87,23 @@ Outputs:
   EndUserGroupArn:
     Value: !GetAtt SCEnduserGroup.Arn
     Export:
-      Name: !Sub '${AWS::Region}-${AWS::StackName}-${RoleAndGroupName}-GroupArn'
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-ServiceCatalogEndusers-GroupArn'
   EndUserGroupName:
     Value: !Ref SCEnduserGroup
     Export:
-      Name: !Sub '${AWS::Region}-${AWS::StackName}-${RoleAndGroupName}-GroupName'
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-ServiceCatalogEndusers-GroupName'
   EndUserRoleId:
     Value: !Ref SCEnduserRole.RoleId
     Export:
-      Name: !Sub '${AWS::Region}-${AWS::StackName}-${RoleAndGroupName}-RoleId'
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-ServiceCatalogEndusers-RoleId'
   EndUserRoleArn:
     Value: !GetAtt SCEnduserRole.Arn
     Export:
-      Name: !Sub '${AWS::Region}-${AWS::StackName}-${RoleAndGroupName}-RoleArn'
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-ServiceCatalogEndusers-RoleArn'
   EndUserRoleName:
     Value: !Ref SCEnduserRole
     Export:
-      Name: !Sub '${AWS::Region}-${AWS::StackName}-${RoleAndGroupName}-RoleName'
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-ServiceCatalogEndusers-RoleName'
```

Comment:
should the ones below, `EndUserExpandPolicy` and `EndUserRemoteAccessPolicy` contain `ServiceCatalogEndusers` in export name as well?

Reply:
No, those are for the policy resources. They are not exports that are from the role or group.